src/python/pants/util/strutil.py

```diff
@@ -238,7 +238,9 @@ def softwrap(text: str) -> str:
             width, use trailing backlashes to line-continue the line. Because we squash multiple
             spaces, this will "just work".)
     """
-    # If callers didnt use a leading "\" thats OK.
+    if not text:
```

Comment:
I was hitting this bug in the new test code, turns out we were calling softwrap() on an empty string in some cases, and it would crash on that.

Reply:
I find it weird that we'd be calling it on an empty string, but okie dokie,

Reply:
I mean it shouldn't crash if we do though, right?

Reply:
Of course. What was the crash btw?